main-project/src/boards/entity/board.entity.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  DeleteDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+
+// fk없음, entity취합 후 생성예정
+@Entity()
+export class Board extends BaseEntity {
```

Comment:
```suggestion
@Entity('boards')
export class Board extends BaseEntity {
```
엔티티에 테이블 이름을 넣어주는 게 보다 명시적인 코드입니다!